app/presenters/setup_presenter.rb

```diff
@@ -1,9 +1,10 @@
 class SetupPresenter
-  attr_reader :current_user, :user_fully_authenticated
+  attr_reader :current_user, :user_fully_authenticated, :user_opted_remember_device_cookie
 
-  def initialize(current_user, user_fully_authenticated)
+  def initialize(current_user, user_fully_authenticated, user_opted_remember_device_cookie)
```

Comment:
Since it might be easy to flip the two arguments since they're both booleans, what do you think of switching these to keywords arguments?